<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StartupDialog</class>
 <widget class="QDialog" name="StartupDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>408</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Starting ADMS...</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
	background-color: rgb(16, 22, 24);
	color: rgb(230, 230, 230);
}

QPushButton {
	border: 1px solid rgb(176, 176, 176);
	border-radius: 2px;
	min-width: 50
}

QPushButton:pressed {
    background-color: rgb(102, 102, 102);
}

QPushButton:default {
    border-color: rgb(102, 255, 255); 
}

QComboBox {
	border: 1px solid rgb(176, 176, 176);
	border-radius: 2px;
	min-width: 50
}

QCheckBox {
	min-width: 50
}

QDateTimeEdit {
	border: 1px solid rgb(176, 176, 176);
	border-radius: 2px;
	min-width: 50
}

QSpinBox {
	border: 1px solid rgb(176, 176, 176);
	border-radius: 2px;
	min-width: 50
}</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0,0,0,0,0,0,0,0,0">
   <item>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>ADMS Chart Visualizer</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Connection</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="connectionBox" native="true">
     <layout class="QFormLayout" name="formLayout">
      <property name="labelAlignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="experimentLabel">
        <property name="text">
         <string>Experiment:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0">
        <item>
         <widget class="QLabel" name="pathEdit">
          <property name="text">
           <string>Path</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="setPathButton">
          <property name="text">
           <string>Find</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="serverLabel">
        <property name="text">
         <string>Server:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="5,0,0">
        <item>
         <widget class="QComboBox" name="serverBox">
          <property name="editable">
           <bool>true</bool>
          </property>
          <item>
           <property name="text">
            <string>tcp://esif-relay-test.hpc.nrel.gov</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>tcp://esif-relay.hpc.nrel.gov</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Port:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="portBox">
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
          <property name="minimum">
           <number>1024</number>
          </property>
          <property name="maximum">
           <number>9999</number>
          </property>
          <property name="value">
           <number>5020</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line_2">
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Display Options</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <property name="labelAlignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="sampleRateLabel">
        <property name="text">
         <string>Resample Rate:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="sampleRateSpinBox">
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="suffix">
         <string>Hz</string>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>60</number>
        </property>
        <property name="value">
         <number>30</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="historyLabel">
        <property name="text">
         <string>Chart History:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QTimeEdit" name="historyTimeEdit">
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="currentSection">
         <enum>QDateTimeEdit::MinuteSection</enum>
        </property>
        <property name="displayFormat">
         <string>m 'm' ss 's'</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="overrideTimeLabel">
        <property name="text">
         <string>Override Time</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QCheckBox" name="overrideTimeCheckBox">
        <property name="toolTip">
         <string>Ignore simulation time, replacing it with a local timer.</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line_3">
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Other</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="openLogButton">
     <property name="text">
      <string>Open Log Directory</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>StartupDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>StartupDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
